src/sentry/options/defaults.py

```diff
@@ -87,6 +87,14 @@
 register("mail.mailgun-api-key", default="", flags=FLAG_ALLOW_EMPTY | FLAG_PRIORITIZE_DISK)
 register("mail.timeout", default=10, type=Int, flags=FLAG_ALLOW_EMPTY | FLAG_PRIORITIZE_DISK)
 
+# Hybrid Cloud
+register(
+    "hybrid-cloud.enable-customer-domains",
+    default=False,
+    type=Bool,
+    flags=FLAG_ALLOW_EMPTY | FLAG_PRIORITIZE_DISK,
+)
```

Comment:
How will this be used differently than the feature flag would be?

Reply:
@markstory When I was making changes to login flows, I wanted to have dedicated code paths for customer domains in cases when the request is un-authenticated. Feature flags won't work here.

In addition, I'd like some changes I'm making not unintentionally impact any self-hosted users.